galaxy_ng/app/api/utils.py

```diff
@@ -53,3 +58,31 @@ def parse_collection_filename(filename):
         raise ValueError(f"Expected version to be max length of {MAX_LENGTH_VERSION}")
 
     return CollectionFilename(namespace, name, version)
+
+
```

Comment:
First thought with this stuff was maybe add a galaxy_ng/app/common/unix_socket_http_adapter.py or whatever.

But I'm not sure there is really any place currently that is an obvious place, so /api/utils.py is probably as good as any.

I would be tempted to split it to it's own file just to keep the list of imports small, but doesn't matter a ton.